case_studies/redshift_estimation/prior.py

```diff
@@ -15,18 +15,18 @@ class RedshiftPrior(CatalogPrior):
     def __init__(
         self,
         *args,
-        redshift_min: float = 0.0,
-        redshift_max: float = 1.0,
+        redshift_alpha: float = 0.0,
```

Comment:
I cherry-picked this file and and merged it into my branch; I left the Uniform prior intact though and added this as a separate class. The user can use either by specifying in the config, and I want to leave the Uniform one in there for testing purposes for now.